### ISSUE TITLE
Code Quality: Introduce Directory.build.props for shared properties

### ISSUE THREAD

**Issue by 0x5bfa** [files-community/Files#14826]
### Description

as the title.
Usage: https://github.com/microsoft/devhome/blob/main/Directory.Build.props

### Concerned code

- csproj and vcxproj files

### Gains

- Reduced lines.
- Avoid forgetting to add required properties
- Avoid duplications

### Requirements

- Add Directory.build.props in the root
- Remove diplications

**Comment by yaira2** [comment]
Can we do this in stages, and start with the duplicate props first?

**Comment by 0x5bfa** [comment]
Yeah starting from
```xml
<Platforms>x86;x64;arm64</Platforms>
<RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
```
would be good

**Comment by yaira2** [comment]
@0x5bfa are we still on board for this? If I recall correctly, this was causing some configuration issues.

**Comment by 0x5bfa** [comment]
Yeah this has great benefits but I think we can slowly adding properties to the shared one, like adding one by one and loading solution step by step on local.
